tests/test_main.py

```diff
@@ -2,6 +2,7 @@
 import ast
 import inspect
 
+from app import main
```

Comment:
The import statement for `main` from `app` is not used in the code. Consider removing it to adhere to the DRY principle and avoid unnecessary imports.